### ISSUE TITLE
Error 118 in Refprop v9.1 after changing mixing parameters for R-32/R-1234yf

### ISSUE THREAD

**Issue by inamdarharshad** [usnistgov/REFPROP-issues#321]
![Refprop error when selecting pre-defined mixtures](https://user-images.githubusercontent.com/22446241/88494875-848e4780-cf7d-11ea-95c3-0393cfd3a6bc.PNG)

### Steps to Reproduce

1. I had originally edited the HMX.BNC file in my Refprop directory to add interaction parameters for R-32/R-1234yf using values provided to me by a refrigerant producer.
2. I re-edited the HMX.BNC file in my Refprop directory to add updated interaction parameters for R-32/R-1234yf using values given to me by same refrigerant producer.
3. I was told that the new interaction parameters were same as those in Refprop v10.0. 
4. There was also a change in the mixing model from KW0 to KW4. 

**Expected behavior:** I expected to be able to re-calculate a spreadsheet where I calculate saturation densities of blends containing R-32 an R-1234yf.

**Actual behavior:** Error message 118 is displayed as shown in the screenshot when I open Refprop GUI and try to select the blend. Spreadsheet is broken as well. Other blends seem to not work either. (Blends where I added mixing model and interaction parameters myself. So, R-410A and R-407C work.)

### Versions

**REFPROP Version:** v9.1  
**Operating System and Version:** Windows 10 Enterprise  
**Access Method:** GUI and Microsoft Excel spreadsheet

**Comment by inamdarharshad** [comment]
I forgot to add: 
5. I also added the LJ6 mixture model in the mixture model specification section after the KWB model and before the LIN model.

**Comment by EricLemmon** [comment]
The only way we can help you is if you send me the hmx.bnc file you modified.  I'll then look at it and tell you what is wrong or missing.  If there is any proprietary material in there, just change the numbers so that they are totally different and then you can change them back once I send the file back to you.  My email is eric.lemmon@nist.gov (ignore my out-of-office message, but do leave a message here that you sent it so that I know to check).

**Comment by inamdarharshad** [comment]
Thank you Dr. Lemmon. I emailed you my HMX.BNC file.

**Comment by inamdarharshad** [comment]
Dr. Lemmon, my issue is resolved. 

To help anyone reading this thread: when I added a mixing model and interaction parameters for custom pairs of fluids, I did not follow the instruction to begin the mixing model identifier at column 27 of the HMX.BNC file. I had pressed tab multiple times to get the cursor to move to the right before copying and pasting the mixing model identifier and interaction parameters obtained from said refrigerant producer. When Dr. Lemmon replaced my tab spaces with actual spaces, my problem was resolved.

Thank you, Dr. Lemmon. It is okay to close this issue now.
